tools/python_api/test/test_scan_pandas.py

```diff
@@ -0,0 +1,52 @@
+import sys
+import numpy as np
+import pandas as pd
+import datetime
+
+sys.path.append('../build/')
+import kuzu
+
+
+def test_scan_pandas(get_tmp_path):
+    db = kuzu.Database(get_tmp_path)
+    conn = kuzu.Connection(db)
+    data = {
+        'BOOL': [True, False, True, False],
+        'UINT8': np.array([1, 2, 3, 4], dtype=np.uint8),
+        'UINT16': np.array([10, 20, 30, 40], dtype=np.uint16),
+        'UINT32': np.array([100, 200, 300, 400], dtype=np.uint32),
+        'UINT64': np.array([1000, 2000, 3000, 4000], dtype=np.uint64),
+        'INT8': np.array([-1, -2, -3, -4], dtype=np.int8),
+        'INT16': np.array([-10, -20, -30, -40], dtype=np.int16),
+        'INT32': np.array([-100, -200, -300, -400], dtype=np.int32),
+        'INT64': np.array([-1000, -2000, -3000, -4000], dtype=np.int64),
+        'FLOAT_32': np.array([-0.5199999809265137, float("nan"), -3.299999952316284, 4.400000095367432],
+                             dtype=np.float32),
+        'FLOAT_64': np.array([5132.12321, 24.222, float("nan"), 4.444], dtype=np.float64),
+        'datetime_microseconds': np.array([
+            np.datetime64('1996-04-01T12:00:11.500001'),
+            np.datetime64('1981-11-13T22:02:52.000002'),
+            np.datetime64('1972-12-21T12:05:44.500003'),
+            np.datetime64('2008-01-11T22:10:03.000004')
+        ]).astype('datetime64[us]'),
+        'timedelta_nanoseconds': [
+            np.timedelta64(500000, 'ns'),
+            np.timedelta64(1000000000, 'ns'),
+            np.timedelta64(2500000000, 'ns'),
+            np.timedelta64(3022000000, 'ns')
+        ]
+    }
+    df = pd.DataFrame(data)
+    results = conn.execute("CALL READ_PANDAS('df') RETURN *")
```

Comment:
Let's add another test to run expression evaluator after READ_PANDAS, e.g. `CALL ... WHERE ... RETURN INT32 + 1`